admin/src/test/java/com/exadel/frs/UserServiceTestIT.java

```diff
@@ -86,6 +92,53 @@ void getEnabledUserByEmailThrowsExceptionIfUserIsDisabled() {
         assertThrows(UserDoesNotExistException.class, () -> userService.getEnabledUserByEmail(DISABLED_USER_EMAIL));
     }
 
+    @Test
+    void getUserByEmailReturnsUser() {
+        createUser(USER_EMAIL);
+
+        val user = userService.getUser(USER_EMAIL);
+
+        assertThat(user).isNotNull();
+    }
+
+    @Test
+    void getUserByEmailThrowsExceptionIfNoUser() {
+        assertThrows(UserDoesNotExistException.class, () -> userService.getUser(USER_EMAIL));
+    }
+
+    @Test
+    void getUserByGuidReturnsUser() {
+        createUser(USER_EMAIL);
+
+        val createdUser = userRepository.findByEmail(USER_EMAIL).get();
+
+        val user = userService.getUserByGuid(createdUser.getGuid());
+
+        assertThat(user).isNotNull();
+    }
+
+    @Test
+    void getUserByGuidThrowsExceptionIfNoUser() {
+        assertThrows(UserDoesNotExistException.class, () -> userService.getUserByGuid(USER_GUID));
```

Comment:
replace all from assertThrows to org.assertj.core.api.Assertions.assertThatThrownBy method call please